crowbar_framework/app/assets/stylesheets/content/content.scss

```diff
@@ -43,6 +33,47 @@
       margin-bottom: 0;
     }
   }
+
+  .panel-plain {
+    @include clearfix;
+  }
+
+  .panel-footer {
+    .progress {
+      background-color: lighten(#f5f5f5, 50%);
+      margin: 6px 0;
+    }
+  }
+}
+
+.wizard {
+  .wizard-nav {
+    background-color: #f5f5f5;
+    margin: 0;
+    padding: 0;
+
+    li {
+      line-height: 40px;
+
+      &.active {
+        a {
+          background-color: #08c;
```

Comment:
Color literals like `#08c` should only be used in variable declarations; they should be referred to via variable everywhere else.